DEVELOPMENT.md

```diff
@@ -34,37 +33,27 @@ The `bootstrap` task will install all of the necessary tools used for developmen
 
 Configuration for running when developing Flipt can be found at `./config/local.yml`. To run Flipt with this configuration, run:
 
-`task server` or `task dev`
+`./bin/flipt --config ./config/local.yml`
 
 ## Changes
 
-Changing certain types of files such as the proto, ui or documentation files require re-building before they will be picked up in new versions of the binary.
+Changing certain types of files such as the proto or ui files require re-building before they will be picked up in new versions of the binary.
 
 ### Updating .proto Files
 
-After changing `flipt.proto`, you'll need to run `task build:proto`. This will regenerate the following files:
+After changing `flipt.proto`, you'll need to run `mage proto`. This will regenerate the following files:
 
 - `rpc/flipt/flipt.pb.go`
 - `rpc/flipt/flipt_grpc.pb.go`
 - `rpc/flipt/flipt.pb.gw.go`
```

Comment:
Bit outside this PR. However, should we adjust this to just say something like:

```
This will generate the revelant Go definitions into the `rpc/flipt` directory.
```

Now that we also generate files in `rpc/flipt/meta` and `rpc/flipt/auth`.